### PR TITLE
fix bot blocked position on export log

### PR DESCRIPTION
**Pull request by heatherlogan-scottlogic**
## Description

Missed out BOT_BLOCKED in getMessageStyle case stmt. 🤷‍♂️ Also added a wee Bot (blocked): for clarity:

## Screenshots

image

## Notes

- Bullet point list with
- any notable code changes
- or explanations regarding this PR

## Concerns

- Bullet point list
- with any questions
- or concerns about this PR

## Checklist

Have you done the following?

- [x] Linked the relevant Issue
- [ ] Added tests
- [x] Ensured the workflow steps are passing

### CODE REVIEW

**Review by gsproston-scottlogic — APPROVED** [review]
hyuge 🙆

File: frontend/src/components/ExportChat/ExportChatMessage.tsx [hunk 1]
```diff
@@ -71,6 +72,7 @@ function getMessageStyle(type: CHAT_MESSAGE_TYPE) {
 			return styles.chatBoxInfo;
 		case CHAT_MESSAGE_TYPE.USER:
 			return styles.chatBoxMessage;
+		case CHAT_MESSAGE_TYPE.BOT_BLOCKED:
```
Comment (pmarsh-scottlogic) [inline frontend/src/components/ExportChat/ExportChatMessage.tsx:75]: I don't quite get what's going on with these empty case statements. How does it behave? Could we write it explicitly?

**Review by pmarsh-scottlogic — APPROVED** [review]
good catch :)

one comment but it's not really to do with the ticket so happy to approve

File: frontend/src/components/ExportChat/ExportChatMessage.tsx (hunk 1 above)
Comment (gsproston-scottlogic) [reply frontend/src/components/ExportChat/ExportChatMessage.tsx:75]: Empty case statements just drop down to the next case statement. It's fairly common to see a "fallthrough" comment in these cases, so we could do that.

```typescript
case CHAT_MESSAGE_TYPE.BOT_BLOCKED:
        // fallthrough
case CHAT_MESSAGE_TYPE.BOT:
	return styles.chatBoxMessageBot;
```

Comment (pmarsh-scottlogic) [reply frontend/src/components/ExportChat/ExportChatMessage.tsx:75]: so 
```typescript
case CHAT_MESSAGE_TYPE.BOT_BLOCKED:
        // fallthrough
case CHAT_MESSAGE_TYPE.BOT:
	return styles.chatBoxMessageBot;
```
is the same as
```typescript
case CHAT_MESSAGE_TYPE.BOT_BLOCKED || CHAT_MESSAGE_TYPE.BOT:
	return styles.chatBoxMessageBot;
```
?

I guess it's more aesthetically pleasing this way. If this is a common thing to do then I'm happy with it!

Comment (gsproston-scottlogic) [reply frontend/src/components/ExportChat/ExportChatMessage.tsx:75]: Sort of, except the latter doesn't quite work. It's sort of the same as doing 
`if (type === (CHAT_MESSAGE_TYPE.BOT_BLOCKED || CHAT_MESSAGE_TYPE.BOT))`
which would be like doing
`if (type === false)`
which isn't what you want.

With switch statements, if you want the same result for multiple values, yes the former is the way to do it 😃

Comment (pmarsh-scottlogic) [reply frontend/src/components/ExportChat/ExportChatMessage.tsx:75]: Ahhh, right! Thanks for educating me :)

**Pull request merged**